source/guides/clear/clear-linux-dissector.rst

```diff
@@ -0,0 +1,302 @@
+
+  .. code-block:: bash
+
+      docker-compose run --rm layersapp /opt/layerindex/layerindex/tools/import_clear.py -d -p /opt/dissector -o /opt/sources -b <branchname> -r 31380
+
+Import |CL| Data via GUI
+------------------------
+
+#. In a browser, navigate to the data import page at
+   https://localhost:8081/layerindex/comparison/import/.
+
+   .. figure:: ../../_figures/clear-linux-dissector/distro-data-import.png
+      :scale: 45%
+      :alt: Distro Data Import page
+
+      Figure 1: Distro Data Import page
+
+#. The first time you import |CL| data, you will need to create a new comparison
+   branch. The branch name you provide will be used in the GUI during compare
+   to refer to this imported release. You can choose to update your branch in
+   later imports.
+
+#. To import the latest |CL| release, check the :guilabel:`Get latest` box.
+
+#. To import a specific release, enter the release number in the
+   :guilabel:`Release` field.
+
+#. Click :guilabel:`IMPORT` and you will be taken to a page that displays the task
+   status. It will take a long time to upload the |CL| data.
+
+#. Once the |CL| data is uploaded, you will see a page similar to the following:
+
+   .. figure:: ../../_figures/clear-linux-dissector/distro-data-import-final.png
+      :scale: 45%
+      :alt: Finished distribution data import
+
+      Figure 2: Finished distribution data import
+
+Import Yocto Project Data
+=========================
+
+Yocto Project data is imported from https://layers.openembedded.org.
+
+Import Yocto Project Data via CLI
+---------------------------------
+
+To import the latest Yocto Project data from the master branch:
+
+* Run the following command:
+
+  .. code-block:: bash
+
+      docker-compose run --rm layersapp /opt/layerindex/layerindex/tools/import_layers.py https://layers.openembedded.org
+
+To import Yocto Project data from a specific branch:
+
+#. First create a branch object in the admin page.
+
+   #. Navigate to https://localhost:8081/admin/layerindex/branch/add/ and enter the
+      branch name in the :guilabel:`Branch name` field.
+
+   #. Enter the corresponding Bitbake branch name (for example "zeus") in the
+      :guilabel:`Bitbake branch` field.
+
+      .. figure:: ../../_figures/clear-linux-dissector/create-yoctoproject-branch.png
+         :scale: 45%
+         :alt: Importing Yocto Project data from branch
+
+         Figure 3: Importing Yocto Project data from branch
+
+
+#. Run the following command to import a specific Yocto Project branch:
+
+   .. code-block:: bash
+
+       docker-compose run --rm layersapp /opt/layerindex/layerindex/tools/import_layers.py https://layers.opnembedded.org -b zeus
+
+To update your Yocto Project branches:
+
+* Run the following command:
+
+  .. code-block:: bash
+
+     docker-compose run --rm layersapp /opt/layerindex/layerindex/update.py
+
+View Distribution Data
+**********************
+View the distribution data of a specific |CL| and Yocto Project distribution
```

Comment:
To help readers understand why this is useful it might be worth stating that this is "package data" or "package information" for the distribution, perhaps we could expand a little on what that means as well.

Reply:
@arbrindle  - can you provide this additional info?

Reply:
View the package data of a specific Clear Linux or Yocto Project distribution. Package data can consist of the package version, description, and license, as well as a link to the package's homepage.